liberty-maven-plugin/pom.xml

```diff
@@ -4,12 +4,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>net.wasdev.wlp.maven</groupId>
+        <groupId>io.openliberty.tools</groupId>
         <artifactId>liberty-maven</artifactId>
         <version>3.1.M1-SNAPSHOT</version>
```

Comment:
This version doesn't look correct.  It should be 3.0.M1-SNAPSHOT or 3.0.M2-SNAPSHOT

Reply:
This version was automatically upgraded by the maven release plugin. I will be changing the version to `3.0-M2-SNAPSHOT` in my next PR.